src/command.ts

```diff
@@ -1,89 +1,114 @@
-/**
- * Module dependencies.
- */
-
-import {EventEmitter} from 'events';
-import _ from 'lodash';
+import {
+  camelCase,
+  isBoolean,
+  isUndefined,
+  isArray,
+  isEmpty,
+  includes,
+  isFunction,
+  isNil
+} from 'lodash';
+
+import { EventEmitter } from 'events';
 import Option from './option';
-import {IAutocompleteConfig} from './types/autocomplete';
-import {ICommand, IVorpal} from './types/types';
+import { AutocompleteConfig } from './autocomplete';
 import util from './util';
+import Vorpal from './vorpal';
+
+export interface Args {
+  [key: string]: string | string[] | object | undefined;
```

Comment:
Would bé interesting to have a (local?) type alias for thé values. (I mean for thé `s|s[]|o|u`